modin/core/io/text/excel_dispatcher.py

```diff
@@ -153,7 +153,7 @@ def _read(cls, io, **kwargs):
             if index_col is not None:
                 column_names = column_names.drop(column_names[index_col])
 
-            if not all(column_names):
+            if not all(column_names) or kwargs["usecols"]:
```

Comment:
Previously, there was no processing of `usecols` parameter in dispatcher. However, for the correct calculation of the columns, we can use pandas, just as we do for `read_csv`.